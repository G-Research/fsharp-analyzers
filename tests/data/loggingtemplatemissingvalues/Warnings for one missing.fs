module M

    open Microsoft.Extensions.Logging

    let testlog () =
        use factory = LoggerFactory.Create(fun b -> b.AddConsole() |> ignore)
        let logger: ILogger = factory.CreateLogger("Program")

        logger.Log(LogLevel.Information, "xxx {one} yyy {two}", 23)
