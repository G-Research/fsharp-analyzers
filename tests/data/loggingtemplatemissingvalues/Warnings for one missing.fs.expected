GRA-LOGTEMPLMISSVALS-001 | Warning | (9,8 - 9,67) | The given values in your call to ILogger.Log{Warning, Error, ...} don't match the expected templated args. | []
